providers/openshift.md

```diff
@@ -32,9 +41,146 @@ master_hostname
 node_hostname
 ```
 
-openshift_use_manageiq is set to True to configure the Service Account needed by ManageIQ. there are more like variables like use_metrics to enable other options in OpenShift.
-A more detailed example will be found in inventory/byo/hosts.example.
+openshift_use_manageiq is set to True to configure the Service Account needed
+by ManageIQ. there are more like variables like use_metrics to enable other
+options in OpenShift.  A more detailed example will be found in
+inventory/byo/hosts.example.
+
+```console
+$ ansible-playbook playbooks/byo/config.yml -i path/to/inventory/file
+```
+
+
+### Running with minishift
+
+
+For a simple single host setup for working with the openshift provider locally,
+you can use [`minishift`](https://github.com/minishift/minishift) to run a
+local openshift instance.
+
+
+#### Requirements
+
+* [`minishift`](https://github.com/minishift/minishift) >= 1.0.0.rc1 ([Installation Instructions](https://docs.openshift.org/latest/minishift/getting-started/installing.html#installing-instructions))
+* Virtualization host of your choice (see supported hypervisors [here](https://docs.openshift.org/latest/minishift/getting-started/installing.html#install-prerequisites))
+
+
+#### Quickstart
+
+Since VirtualBox is a consistent hypervisor across all platforms, this guide
+will assume that is being used, and updates to some commands might be
+necessary.  Most commands should be virtualization software agnostic and work
+regardless of the hypervisor.
 
-openshift_use_manageiq is set to True to configure the Service Account needed by ManageIQ. there are more like variables like use_metrics to enable other options in OpenShift.
-A more detailed example will be found in inventory/byo/hosts.example.
-```ansible-playbook playbooks/byo/config.yml -i path/to/inventory/file```
+1. Create an addon directory for manageiq in minishift.  Something like:
+  
+  ```console
+  $ mkdir -p ./minifshift/addons/manageiq
```

Comment:
This is another reason I am pushing this to be added to the core part of the repo as well, since these instructions are a bit derpy...  https://github.com/minishift/minishift/issues/710